vsprojects/grpc_csharp_ext/grpc_csharp_ext.vcxproj

```diff
@@ -80,6 +80,9 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <TargetName>grpc_csharp_ext</TargetName>
+    <Linkage-grpc_dependencies_zlib>static</Linkage-grpc_dependencies_zlib>
```

Comment:
Does this mean we are planning to always link zlib statically and distribute it as a part of gRP?. This also means that we would need to drop the dependency on https://www.nuget.org/packages/grpc.dependencies.zlib/.